megamek/src/megamek/common/loaders/BLKSupportVTOLFile.java

```diff
@@ -97,9 +97,6 @@ public Entity getEntity() throws EntityLoadingException {
             t.setFuelTonnage(dataFile.getDataAsDouble("fuel")[0]);
         }
         int engineFlags = Engine.TANK_ENGINE | Engine.SUPPORT_VEE_ENGINE;
-        if (t.isClan()) {
```

Comment:
How come this block is removed here but added to in the other files?

Reply:
Support vehicle engines don't distinguish between IS and Clan, so this block serves no purpose.